src/frontend/tsx/Chessboard.tsx

```diff
@@ -196,6 +191,17 @@ export function Chessboard () {
       fen: (displayIndex == null || displayGame.info.moves == null)
         ? chess.fen()
         : displayGame.info.moves[displayIndex].fen,
+      lastMove: (displayGame == null || displayGame.info.moves == null)
+        ? null
+        : (displayIndex == null)
+          ? [
+            displayGame.info.moves[displayGame.info.moves.length - 1].from,
```

Comment:
These lines appear to be causing a bug on a fake ship. When selecting an active game, the screen goes blank and the JS console gives me the error: "Uncaught TypeError: t.info.moves[(t.info.moves.length - 1)] is undefined"

Reply:
I've had the "screen goes blank" bug a few times before and in my experience it's always because the frontend is trying to render something that's `undefined` or `null`.